svthalia/concrexit
# Student participation page overhaul

Closes #2621 

### Summary
- Updated the text on the program committee per request of the program committee (see issue #2621 )
- Moved the student participation page from education to singlepages because it is a single static page and now it is in the same place as the student well-being page, which is very similar to it. Other related files have been changed accordingly.
- Changed the layout of the information on the student participation page, including clickable squares to webpages in a similar way as on the well-being page. The layout is responsive to screen-size. 
- Included logos of the student participation bodies.

### How to test
Go to education -> student participation page
Check if it works for multiple screen sizes
